homeassistant/components/heos/config_flow.py

```diff
@@ -74,3 +81,24 @@ async def async_step_user(
             data_schema=vol.Schema({vol.Required(CONF_HOST, default=host): host_type}),
             errors=errors,
         )
+
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Allow reconfiguration of entry."""
+        await self.async_set_unique_id(DOMAIN)
```

Comment:
```suggestion
```
don't think you need that here

Reply:
Confirmed with added `assert config_entry.unique_id == 'heos'` in test.